website/intro/getting-started/variables.html.md

```diff
@@ -122,8 +122,10 @@ values are not saved, but this provides a convenient workflow when getting
 started with Terraform. UI Input is not recommended for everyday use of
 Terraform.
 
--> **Note**: UI Input is only supported for string variables. List and map
-variables must be populated via one of the other mechanisms.
+-> **Note**: In Terraform versions 0.11 and earlier, UI Input is only supported
+for string variables. List and map variables must be populated via one of the
+other mechanisms. Terraform 0.12 introduces the ability to populate complex
+variable types from the UI prompt.
```

Comment:
Huh... I don't think we actually _intentionally_ changed this behavior, but I guess the new behavior is useful so we might as well document it and keep it!